Formula/p/pan.rb

```diff
@@ -0,0 +1,62 @@
+class Pan < Formula
+  desc "Usenet newsreader that's good at both text and binaries"
+  homepage "https://pan.rebelbase.com"
+  url "https://gitlab.gnome.org/GNOME/pan/-/archive/v0.154/pan-v0.154.tar.bz2"
+  sha256 "440317954df7217689100df3dfb68865770f5aed1b8ed2b45432d771bb80a8c9"
+  license "GPL-2.0-only"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "itstool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "adwaita-icon-theme"
+  depends_on "cairo"
+  depends_on "enchant"
+  depends_on "gdk-pixbuf"
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "gmime"
+  depends_on "gnutls"
+  depends_on "gtk+3"
+  depends_on "gtkspell3"
+  depends_on "harfbuzz"
+  depends_on "pango"
+
+  def install
+    # use brew name for gtk3 version of tool update-icon-cache
+    inreplace  "pan/icons/Makefile.am", "gtk-update-icon-cache", "gtk3-update-icon-cache"
+
+    # fix libiconv linking https://gitlab.gnome.org/GNOME/pan/-/issues/171
+    ENV.append "LDFLAGS", "-liconv" if OS.mac?
+    ENV.append "CXXFLAGS", "-std=c++11"
+
+    system "NOCONFIGURE=1 ./autogen.sh"
+    system "./configure", *std_configure_args,
+                          "--disable-silent-rules",
+                          "--with-gnutls"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
```

Comment:
You could probably return early here on Linux, since testing GTK doesn't work too well in a headless container.